packages/block-editor/src/components/block-inspector/style.scss

```diff
@@ -11,10 +11,8 @@
 	}
 
 	.components-base-control {
-		margin-bottom: #{ $grid-unit-30 };
-
-		&:last-child {
-			margin-bottom: $grid-unit-10;
```

Comment:
While we work on this, could we also find a way to move away from using `.components-base-control` at all?

Apart from the fact that is a private API of the components package and that it makes the UI fragile to future changes, it also doesn't work for controls that are not wrapped in base control.

Maybe we can introduce a `block-editor` package-specific classname instead (ie. `block-editor-inspector-controls__control` or something) and apply it where relevant? Or switch to `VStack` or similar, where the vertical spacing is defined on the parent and not the children?

Reply:
The pathway I have in mind at the moment is:

1. Add a `__next` prop and/or data attribute so certain container components can opt out of the automatic margin-bottom on BaseControl. Ideally we'd expose the prop only on `InspectorControls` and not on any `@wordpress/components` components.
2. Figure out a standard way we can recommend to WP devs for spacing out their inspector controls. Is it VStack (not ideal because the current default `spacing` is 8px instead of the 16px we want in the inspector)? Is it a new, dedicated layout component (#43423)? To be decided.
3. Migrate everything in the codebase.
4. Deprecate the automatic margin-bottom on BaseControl.

Another long journey 😇

Reply:
Extracted to a tracking issue: #65191

Reply:
> Hmm, interesting. @oandregal Are they meant for the block inspector at all? My first impression is that it could be used for some cases, but it wouldn't cover everything.

I presume any place that uses a complex form is a potential consumer of DataForm. I don't think it is ready for replacing inspector controls yet.